FastlaneExample/fastlane/Fastfile

```diff
@@ -0,0 +1,94 @@
+# https://github.com/fastlane/fastlane/tree/master/fastlane/docs
+# All available actions: https://docs.fastlane.tools/actions
+# can also be listed using the `fastlane actions` command
+# To edit any of the Enviroment variables please do so from the .env files
+
+fastlane_version "2.14.2"
+default_platform :ios
+
+platform :ios do
+
+  ### Define Enviroment variables ###
+  ENV["SCHEME_BETA"] = "Beta"
+
+  desc "Runs all the tests"
+  lane :test do
+    scan(scheme: 'FastlaneExampleTests')
+  end
+
+  desc "Runs all the UI tests"
+  lane :ui_tests do
+    scan(scheme: 'UITests')
```

Comment:
can we use the "descriptive" names here? I always find fastlane's overly metaphorical naming confusing and hard to know what it is doing, and clearly they did too, so now they have [more sensible naming](https://github.com/fastlane/fastlane/pull/10939). For example:

`scan` => `run_tests`
`gym` => `build_ios_app`
`deliver` => `upload_to_app_store`